examples/cpu-hello-world/main.cpp

```diff
@@ -133,18 +129,18 @@ static SlangResult _innerMain(int argc, char** argv)
     // Get the 'shared library' (note that this doesn't necessarily have to be implemented as a shared library
     // it's just an interface to executable code).
     ComPtr<ISlangSharedLibrary> sharedLibrary;
-    SLANG_RETURN_ON_FAIL(spGetEntryPointHostCallable(slangRequest, 0, 0, sharedLibrary.writeRef()));
+    SLANG_RETURN_ON_FAIL(spGetTargetHostCallable(slangRequest, 0, sharedLibrary.writeRef()));
 
     // Once we have the sharedLibrary, we no longer need the request
     // unless we want to use reflection, to for example workout how 'UniformState' and 'UniformEntryPointParams' are laid out
     // at runtime. We don't do that here - as we hard code the structures. 
     spDestroyCompileRequest(slangRequest);
 
     // Get the function we are going to execute 
+    const char entryPointName[] = "computeMain";
     CPPPrelude::ComputeFunc func = (CPPPrelude::ComputeFunc)sharedLibrary->findFuncByName(entryPointName);
     if (!func)
     {
-        spDestroyCompileRequest(slangRequest);
```

Comment:
Why did you remove the cleanup code on the failure path?

Reply:
It is already destroyed in the line above regardless of failure here.